src/main/java/com/ibm/crypto/plus/provider/EdDSAKeyPairGenerator.java

```diff
@@ -102,7 +102,7 @@ public void initialize(AlgorithmParameterSpec params)
     @Override
     public KeyPair generateKeyPair() {
         try {
-            int pub_size = CurveUtil.getPublicCurveSize(curve);
+            int pub_size = CurveUtil.getCurveSize(curve);
```

Comment:
Should we change the name of the variable to `keySize` or something similar, since this is not just the size of the public key?